### PR TITLE
Sync on sharing a directory that has a dat in it already

### LINKED ISSUE #535: 'dat previously downloaded' confusing
I don't understand 'dat previously downloaded' Run `dat <key>` to resume.

Issues I see:
1. the key that's printed isn't a hex string. Is this the same hash we had before? It should probably print in hex, in any case.
2. If its the same key, why can't it just resume for me?

### PR DESCRIPTION
**Pull request by karissa**
It used to give some output that told the user to run a manual command to sync. Now it'll just sync for them.

fixes #535

### CODE REVIEW

**Comment by karissa** [comment]
cc @yoshuawuyts

File: commands/download.js [hunk 1]
```diff
@@ -1,14 +1,10 @@
 var chalk = require('chalk')
 var prettyBytes = require('pretty-bytes')
-var Dat = require('dat-js')
 var logger = require('status-logger')
 var speedometer = require('speedometer')
 var ui = require('../lib/ui')
 
-module.exports = function (args) {
-  if (args && args.exit) args.upload = false
```
Comment (joehand) [inline commands/download.js]: We lost this setting. When `--exit` was specified, the download wouldn't upload data. 

I am fine getting rid of if if you think so too.

**Comment by joehand** [comment]
Looks good, just that one comment. I'm fine merging + releasing as-is though.

File: commands/download.js (hunk 1 above)
Comment (karissa) [inline commands/download.js]: yeah i think its fine to get rid of.

also i'm not sure what `args.upload = false` was doing or how that was related

Comment (joehand) [inline commands/download.js]: `args.upload = false` prevented re-seeding data. They are not really related... 

I think the thought with `--exit` was you want to download as quickly as possible, so uploading would slow that down.

Comment (karissa) [inline commands/download.js]: ah ok

**Comment by Neurone** [comment]
Since this merge there's an error that affect the use of the command `dat ` without parameters. It should return the help, like v11.4.0, but it now returns:
```
/usr/lib/node_modules/dat/node_modules/dat-js/index.js:19
  if (!opts.dir) throw new Error('dir option required')
                 ^

Error: dir option required
    at new Dat (/usr/lib/node_modules/dat/node_modules/dat-js/index.js:19:24)
    at Dat (/usr/lib/node_modules/dat/node_modules/dat-js/index.js:17:38)
    at run (/usr/lib/node_modules/dat/bin/cli.js:54:13)
    at Object.<anonymous> (/usr/lib/node_modules/dat/bin/cli.js:49:32)
    at Module._compile (module.js:570:32)
    at Object.Module._extensions..js (module.js:579:10)
    at Module.load (module.js:487:32)
    at tryModuleLoad (module.js:446:12)
    at Function.Module._load (module.js:438:3)
    at Module.runMain (module.js:604:10)
```
The problem is in the use of dat-js in run method of cli.js. Dat-js require dir parameter. In fact, if you use:

`dat --dir this_is_a_test`

it correctly returns the help screen. I tested 11.5.0, 11.5.1 and 11.5.2 and all are affected.

**Pull request merged**